test/pylib/manager_client.py

```diff
@@ -135,10 +136,17 @@ async def server_add(self) -> str:
         logger.debug("ManagerClient added %s", server_id)
         return server_id
 
-    async def server_remove(self, server_id: str) -> None:
+    async def server_remove(self, to_remove_ip: str) -> None:
```

Comment:
`server_remove` and `remove_node` should be a single function.
`server_remove` on its own serves no purpose.

Reply:
Removed old and unimplemented `server_remove` and implemented a proper one in https://github.com/scylladb/scylladb/commit/7cc3aedb8d083a86de5cf4d92fd4c3190af4413d